profiles/killbill/src/test/java/org/killbill/billing/jaxrs/TestAccount.java

```diff
@@ -54,9 +55,8 @@
 import org.testng.annotations.BeforeMethod;
 import org.testng.annotations.Test;
 
-import com.google.common.collect.ImmutableList;
+// FIXME-1615 : killbill-client-java : RequestOptions using MultiMap
```

Comment:
If you need to update the generator https://github.com/killbill/killbill-swagger-coden (but I don't think so), you can ping @reshmabidikar -- she's the new swagger expert 😅